app/javascript/react/screens/App/App.scss

```diff
@@ -1,2 +1,3 @@
+@import 'patternfly/dist/sass/patternfly/color-variables';
```

Comment:
This breaks gaprindashvili ui-classic travis:

```
     # --- Caused by: ---
     # Sass::SyntaxError:
     #   File to import not found or unreadable: patternfly/dist/sass/patternfly/color-variables.
```

(https://travis-ci.org/ManageIQ/manageiq-ui-classic/builds/391256750)

There is no `patternfly`, since neither v2v nor ui-classic depend on any such package now.

This should be `patternfly-sass/dist/...` - fixed in #410.